control-plane/tests/test_service_persistence_action_reconciliation_review_surfaces.py

```
@@ -0,0 +1,1236 @@
+        self.assertEqual(queue_snapshot["total_records"], 1)
+        self.assertEqual(
+            queue_snapshot["records"][0]["current_action_review"]["action_request_id"],
+            second_request.action_request_id,
+        )
+        self.assertLessEqual(store.list_calls, 6)
+        self.assertEqual(store.reconciliation_list_calls, 2)
+        self.assertNotEqual(first_request.action_request_id, second_request.action_request_id)
+    def test_service_action_review_surfaces_prefer_live_approval_and_execution_records(
+        self,
+    ) -> None:
+        store, service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.10)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)